src/api/app/assets/stylesheets/webui/bs5-shims.scss

```diff
@@ -0,0 +1,16 @@
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
```

Comment:
This is defined [upstream in Bootstrap](https://github.com/twbs/bootstrap/blob/d70b5db2e0aee9ea9a7255d28395278876bf53ba/scss/mixins/_breakpoints.scss#L48-L57).